[2024-01-29T03:38:50.478+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: jora_crawler.save_to_s3 scheduled__2024-01-28T12:00:00+00:00 map_index=2 [queued]>
[2024-01-29T03:38:50.562+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: jora_crawler.save_to_s3 scheduled__2024-01-28T12:00:00+00:00 map_index=2 [queued]>
[2024-01-29T03:38:50.567+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-01-29T03:38:50.708+0000] {taskinstance.py:2191} INFO - Executing <Mapped(_PythonDecoratedOperator): save_to_s3> on 2024-01-28 12:00:00+00:00
[2024-01-29T03:38:50.845+0000] {standard_task_runner.py:60} INFO - Started process 394 to run task
[2024-01-29T03:38:50.905+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'jora_crawler', 'save_to_s3', 'scheduled__2024-01-28T12:00:00+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/crawler/jora_cralwer.py', '--cfg-path', '/tmp/tmpix1za67e', '--map-index', '2']
[2024-01-29T03:38:50.985+0000] {standard_task_runner.py:88} INFO - Job 121: Subtask save_to_s3
[2024-01-29T03:38:51.698+0000] {task_command.py:423} INFO - Running <TaskInstance: jora_crawler.save_to_s3 scheduled__2024-01-28T12:00:00+00:00 map_index=2 [running]> on host 5a789d336170
[2024-01-29T03:38:52.703+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='jora_crawler' AIRFLOW_CTX_TASK_ID='save_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-01-28T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-28T12:00:00+00:00'
[2024-01-29T03:38:52.725+0000] {logging_mixin.py:188} INFO - c_crawl
[2024-01-29T03:38:52.738+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-29T03:38:52.847+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=jora_crawler, task_id=save_to_s3, map_index=2, execution_date=20240128T120000, start_date=20240129T033850, end_date=20240129T033852
[2024-01-29T03:38:53.043+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-29T03:38:53.146+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
